AccountTests/HawkHelperTests.swift

```diff
@@ -70,7 +75,9 @@ class HawkHelperTests: XCTestCase {
         let timestamp = Int64(1353832234)
         let nonce = "j4h3g2"
         let extra = "some-app-ext-data"
+        // swiftlint:disable line_length
```

Comment:
Instead of disabling the linter here can you instead use multi-line strings? You can add a \ anywhere you add a newline so the string literal ignores the newline character.
https://www.hackingwithswift.com/sixty/1/3/multi-line-strings

Reply:
Great point, ty! :)